node/src/chain_spec.rs

```diff
@@ -394,61 +394,6 @@ pub fn development_config() -> DevelopmentChainSpec {
 				1000.into(),
 			)
 		},
-		Vec::new(),
-		None,
-		None,
-		None,
-		None,
-		Extensions {
-			relay_chain: "rococo-local".into(), // You MUST set this to the correct network!
-			para_id: 1000,
-		},
-	)
-}
-
-pub fn local_testnet_config() -> DevelopmentChainSpec {
```

Comment:
It was confusing that we have both a local testnet and a development testnet. I don't think that we need this distinction as both chain specs are basically the same. I only kept the testnet definition.